scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -0,0 +1,106 @@
+core.controllers.base_test
```

Comment:
I prefer having the test module names in the ignore list because that better reflects the implementation. For this discussion I'll refer to the non-test python modules as "code modules." We identify which tests to run by test module, not code module, because the test modules are what we list in the shards (we do this because the sharding is primarily about the runtime of the tests). Therefore, I think it makes more sense to have test modules in the exclusion list.

If we were to put code modules in the exclusion list, we might give the impression that we are checking that every code module has an accompanying test module. We do not do this. In fact, there are lots of code modules, particularly in the extensions, that lack test modules. Writing test modules for these is outside the scope of the current project.

Reply:
> I prefer having the test module names in the ignore list because that better reflects the implementation.

I believe this file is going to be edited by anyone who is adding a new test for a given file to have 100% coverage, so I would prefer it to be a list of files/modules that need 100% coverage. 

> We identify which tests to run by test module, not code module because the test modules are what we list in the shards (we do this because the sharding is primarily about the runtime of the tests).

I think it's fine cause we don't expect other contributors (especially new contributors) to edit the shard.json file and the file explains itself that it divides all the test files into different shards.

> We do not do this. In fact, there are lots of code modules, particularly in the extensions, that lack test modules.

That's interesting, I can see the complexity. Considering this is not an ever lasting file in the codebase, I'm fine with having the test-module here let's just make it very clear to developers (in the issue where we are going to list the test coverage task) about the "code-module" they have to look for the listed modules in this file.

Thanks for explaining!